dev/ProgressBar/ProgressBar.xaml

```diff
@@ -65,6 +70,22 @@
                                 <VisualState x:Name="Normal" />
                                 <VisualState x:Name="Determinate" />
                                 <VisualState x:Name="Updating" />
+                                <VisualState x:Name="UpdatingError">
+                                    <Storyboard>
+                                        <Storyboard>
+                                            <ColorAnimation
+                                            Storyboard.TargetName="DeterminateProgressBarIndicator"
+                                            Storyboard.TargetProperty="(Shape.Fill).(SolidColorBrush.Color)"
+                                            To="{Binding Source={ThemeResource SystemControlErrorTextForegroundBrush},Converter={StaticResource SolidColorBrushToColor}}"
```

Comment:
Does
`
To="{Binding Source={ThemeResource SystemControlErrorTextForegroundBrush},Path=Color}"
`
not work?

Reply:
It does to the same degree as converters: It doesn't pick up a resource override. 

I've changed it to path as we should avoid those converters if they are not necessary, thanks for pointing that out.

Reply:
That resource failure lookup seems familiar....and turns out in another issue, we already started speculating if the resource lookup by the `Binding` function is currently broken:

See the two messages starting [here](https://github.com/microsoft/microsoft-ui-xaml/issues/2774#issuecomment-652035802).

Somehow this managed to slip off my mind when it comes to creating a separate issue but this PR serves as a good reminder for that. I would create such an issue if you don't mind?

Reply:
Feel free to create an issue for this. Another issue caused by this, which also slipped my mind, is #3083 which exhibits the same behavior.

I think the issue with themresource lookup inside binding is the following:
The binding is inside WinUI, so the resource probably starts there. It then goes up the chain, and since you load WinUI in your app.xaml, this is the next place where it will look for that resource. Overriding those resources in App.xaml works as expected, but overriding on pages does not work with the classical setup.

Reply:
> Overriding those resources in App.xaml works as expected, but overriding on pages does not work with the classical setup.

In my case linked above, not even the app level was working: 

> It also does not matter if I override these resources on the control level, the page level or the app level. Same effect - my overriden placeholder foreground theme resources will be ignored.

Reply:
Nevermind, with the progressbar, it worked once for some reason but now it doesn't anymore...

Really strange how resourcelookup behaves while inside binding.

Reply:
Does the comment from the other PR about bindings in styles only being applied once explain something here?

Reply:
Yes, that comment is also explaining why this is not working as expected. What do you think should we do with this given that we would need to manually do the binding for it to update correctly?